test/images/cloudbuild.yaml

```
@@ -9,6 +9,22 @@ options:
   substitution_option: ALLOW_LOOSE
   machineType: 'N1_HIGHCPU_8'
 steps:
+  - name: gcr.io/cloud-builders/gcloud
+    entrypoint: 'bash'
+    # NOTE(claudiub): We need to get the ca.pem, cert.pem, key.pem files and put create the
+    # /certs/.docker-1809/, /certs/.docker-1903/, /certs/.docker-1909/ folders, which will contain the files.
+    args:
+    - -c
+    - 'mkdir .docker/windows &&\
+       gcloud secrets versions access latest --project=k8s-infra-prow-build-trusted --secret=windows-remote-docker_ca-pem > .docker-windows/ca.pem &&\
+       gcloud secrets versions access latest --project=k8s-infra-prow-build-trusted --secret=windows-remote-docker_cert-pem > .docker-windows/cert.pem &&\
+       gcloud secrets versions access latest --project=k8s-infra-prow-build-trusted --secret=windows-remote-docker_key-pem > .docker-windows/key.pem &&\
+          cp -r .docker-windows /certs/.docker-1809 && \
+          cp -r .docker-windows /certs/.docker-1903 && \
+          cp -r .docker-windows /certs/.docker-1909'
```

Looks like you have a typo in here, I'm not 100% sure which is right.  The main point of the suggestion is you don't need to quote and && everything
```suggestion
    - |
      mkdir -p .docker-windows
      gcloud secrets versions access latest --project=k8s-infra-prow-build-trusted --secret=windows-remote-docker_ca-pem > .docker-windows/ca.pem
      gcloud secrets versions access latest --project=k8s-infra-prow-build-trusted --secret=windows-remote-docker_cert-pem > .docker-windows/cert.pem
      gcloud secrets versions access latest --project=k8s-infra-prow-build-trusted --secret=windows-remote-docker_key-pem > .docker-windows/key.pem
      cp -r .docker-windows /certs/.docker-1809
      cp -r .docker-windows /certs/.docker-1903
      cp -r .docker-windows /certs/.docker-1909
```